apps/src/templates/teacherDashboard/LoginTypeCard.jsx

```diff
@@ -12,75 +11,78 @@ const styles = {
     // Set width to form a three-column layout on 970px teacher dashboard.
     width: 312,
     // Uniform height, even in different rows
-    height: 240,
-    paddingTop: 15,
-    paddingLeft: 20,
-    paddingRight: 10,
-    paddingBottom: 15,
+    height: 150,
+    paddingTop: 16,
+    paddingLeft: 16,
+    paddingRight: 16,
+    paddingBottom: 16,
     marginBottom: 5,
     borderStyle: 'solid',
     borderWidth: 1,
     borderColor: color.border_gray,
+    borderRadius: 5,
     background: color.white
   },
   title: {
-    paddingTop: 10,
-    paddingBottom: 5,
-    fontSize: 18,
+    paddingBottom: 6,
+    fontSize: 24,
     lineHeight: '18px',
-    fontFamily:'"Gotham 4r", sans-serif',
+    fontFamily:'"Gotham 5r", sans-serif',
     zIndex: 2,
     color: color.dark_charcoal,
-    fontWeight: 'bold'
   },
   subtitle: {
+    paddingBottom: 12,
     fontSize: 14,
     lineHeight: '18px',
     fontFamily:'"Gotham 4r", sans-serif',
     zIndex: 2,
     color: color.dark_charcoal,
-    fontWeight: 'bold'
   },
   description: {
     paddingTop: 10,
     paddingBottom: 5,
-    fontSize: 14,
+    fontSize: 13,
     lineHeight: '18px',
     fontFamily: '"Gotham 4r", sans-serif',
     zIndex: 2,
     color: color.dark_charcoal
   },
-  button: {
-    alignSelf: 'flex-start',
-    marginTop: 20,
-    marginBottom: 10,
-    zIndex: 2,
-  },
 };
 
 class LoginTypeCard extends Component {
   static propTypes = {
     title: PropTypes.string.isRequired,
     subtitle: PropTypes.string,
     description: PropTypes.string.isRequired,
-    buttonText: PropTypes.string.isRequired,
     onClick: PropTypes.func.isRequired,
     className: PropTypes.string,
-    disabled: PropTypes.bool,
+  };
+
+  state = {
+    hover: false,
+  };
+
+  toggleHover = () => {
+    this.setState({ hover: !this.state.hover });
   };
 
   render() {
     const {
       title,
       subtitle,
       description,
-      buttonText,
       onClick,
-      disabled
     } = this.props;
 
+    if (this.state.hover) {
+      styles.card.borderColor = color.dark_charcoal;
+    } else {
```

Comment:
you could simplify this conditional by adding `borderColor: color.border_gray` to `styles.card`, since it's the default, then remove the `else` condition here

Reply:
This didn't work :/